### ISSUE TITLE
Composition Queue Generation Errors

### ISSUE THREAD

**Issue by Sumukh** [Cal-CS-61A-Staff/ok#548]
Running the seed file provides the following traceback when creating composition queues.
```
  File ".../server/app/seed/__init__.py", line 373, in seed
    utils.add_to_grading_queues(assign.key)
  File ".../server/app/utils.py", line 322, in add_to_grading_queues
    seen.add(subm.get().submitter.id())
AttributeError: 'FinalSubmission' object has no attribute 'get'
```

I think this is related to immutablilty tweak from a while back - but I'm not sure (@AlvinWan)

**Comment by alvinwan** [comment]
Euh. I pulled master but couldn't reproduce this either. I must be doing something wrong - what should I do get this error? `source activate_server.sh` with the latest version of master seems to work ok.

**Comment by alvinwan** [comment]
*Never mind. I saw it. Not sure what I did differently. It was a quick fix #551
